### PR TITLE
Feat/document UUID

### PR DESCRIPTION
**Pull request by dominikhorn93**
---
name: Pull request
about: Contribute your changes to the project
title: '[pull-request]'
labels: ''
assignees: ''
---
**Description**
Short description or comments

**Reference**
Issues: https://github.com/it-at-m/digiwf-project/issues/270

**Checklist**

- [x] Acceptance criteria are met
- [x] Pipeline has been run successfully
- [ ] JUnit tests have been run successfully
- [x] Changelog is adjusted
- [ ] [Documentations](https://git.muenchen.de/groups/digitalisierung/-/wikis/Dokumentationen) are completed
- [x] Frontend is tested
- [ ] Created sub-branches are deleted
- [x] Boards are updated ( [Digitalisierung](https://git.muenchen.de/groups/digitalisierung/-/boards) , [Support](https://git.muenchen.de/digitalisierung/digiwf-support) , [Zenhub](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) )
- [ ] feature process is created or extended
- [ ] [git-ops](https://git.muenchen.de/digitalisierung/digiwf-ops) is customized
- [ ] Openshift environments are prepared (Secrets, etc.)

### CODE REVIEW

File: src/lib-components/VMultiFileInput.vue [hunk 1]
```diff
@@ -190,6 +213,24 @@ export default class VMultiFileInput extends Vue {
     return mimetype ? mimetype : "plain/text";
   }
 
+  //generate uuid
```
Comment (martind260) [inline src/lib-components/VMultiFileInput.vue:216]: There is a library "uuid" which can do that.

**Comment by martind260** [comment]
Why didnt you fill out the DoD? Is there something wrong with it?

File: src/lib-components/VMultiFileInput.vue [hunk 2]
```diff
@@ -41,16 +41,13 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Inject, Prop, Vue } from "vue-property-decorator";
```
[the remark this answers is not in the document]
Comment (dominikhorn93) [reply src/lib-components/VMultiFileInput.vue:44]: added the .editorconfig from digitalwf-frontend

File: src/lib-components/VMultiFileInput.vue (hunk 1 above)
Comment (dominikhorn93) [reply src/lib-components/VMultiFileInput.vue]: thanks for the hint. switched to the library

**Comment by dominikhorn93** [comment]
> @dominikhorn93 Why didnt you fill out the DoD? Is there something wrong with it?

was to lazy... adjusted it.

**Review by martind260 — APPROVED** [review]

**Pull request merged**